### ISSUE TITLE
Link to galaxy tool dependencies is dead

### ISSUE THREAD

**Issue by infB** [galaxyproject/galaxy-hub#238]
There are some links all pointing to the ToolDependencies, which are dead.

1. Dead link:
Location: https://github.com/galaxyproject/galaxy/
File:  README.rst 
The provided link is: https://wiki.galaxyproject.org/Admin/Tools/ToolDependencies

2. Dead link:
Location: https://galaxyproject.org/admin/config/tool-dependencies/
The provided link is: https://wiki.galaxyproject.org/Admin/Tools/ToolDependencies

**Comment by pjbriggs** [comment]
+1 I've just encountered this broken link too (I'm trying to find if there is a recommended version of `samtools` to install).

**Comment by bgruening** [comment]
@pjbriggs samtools < 1.0.

**Comment by pjbriggs** [comment]
Thanks @bgruening

**Comment by mvdbeek** [comment]
No, please install the latest samtools. Galaxy can deal with both, but old samtools will fall flat on the nose for long reads.

**Comment by martenson** [comment]
This is related to the faulty conversion form old wiki when we hit `surge protection` while scrapping. 

At least 16 pages are still missing.

https://github.com/galaxyproject/galaxy-hub/search?utf8=%E2%9C%93&q=Surge+protection&type=

**Comment by stephen-kolmann** [comment]
I just experienced this issue too. I was on the admin [Get Galaxy](https://galaxyproject.org/admin/get-galaxy/) page, I clicked a link to send me to the [Tool Dependencies](https://galaxyproject.org/admin/tools/ToolDependencies/) page, but the link is incorrect. The correct tool dependencies URL is https://galaxyproject.org/admin/config/tool-dependencies/.

**Comment by martenson** [comment]
@stephen-kolmann should be fixed by https://github.com/galaxyproject/galaxy-hub/commit/fac7ecb5ea27046fe830afda65d16614aa5cb742
